tests/integration_tests/test_sampler.py

```diff
@@ -1,170 +0,0 @@
-from typing import Callable
-from unittest.mock import patch
-
-import pytest
-
-import optuna
-from optuna.integration import PyCmaSampler
-from optuna.integration import SkoptSampler
-from optuna.samplers import BaseSampler
-from optuna.testing.samplers import FirstTrialOnlyRandomSampler
-
-
-pytestmark = pytest.mark.integration
-
-
-parametrize_sampler = pytest.mark.parametrize(
-    "sampler_class", [optuna.integration.SkoptSampler, optuna.integration.PyCmaSampler]
-)
-
-
-@pytest.mark.parametrize(
-    "sampler_class",
-    [
-        lambda: SkoptSampler(independent_sampler=FirstTrialOnlyRandomSampler()),
-        lambda: PyCmaSampler(independent_sampler=FirstTrialOnlyRandomSampler()),
-    ],
-)
-def test_suggested_value(sampler_class: Callable[[], BaseSampler]) -> None:
-
-    sampler = sampler_class()
-    # direction='minimize'
-    study = optuna.create_study(sampler=sampler, direction="minimize")
-    study.optimize(_objective, n_trials=10, catch=())
-    for trial in study.trials:
-        for param_name, param_value in trial.params.items():
-            distribution = trial.distributions[param_name]
-            param_value_in_internal_repr = distribution.to_internal_repr(param_value)
-            assert distribution._contains(param_value_in_internal_repr)
-
-    # direction='maximize'
-    study = optuna.create_study(sampler=sampler, direction="maximize")
-    study.optimize(_objective, n_trials=10, catch=())
-    for trial in study.trials:
-        for param_name, param_value in trial.params.items():
-            distribution = trial.distributions[param_name]
-            param_value_in_internal_repr = distribution.to_internal_repr(param_value)
-            assert distribution._contains(param_value_in_internal_repr)
```

Comment:
[Note] I found that `test_float` in tests/samlers_test/test_samplers.py checks only 
`minimize`, but I don't think we need to keep this test case only for `CmaSampler` and `SkoptSampler`.

https://github.com/optuna/optuna/blob/58434f930c606f539ff75d0a1cd96c1f82362e9e/tests/samplers_tests/test_samplers.py#L261-L264